packages/broker/test/integration/SubscriptionManager.test.ts

```diff
@@ -28,8 +28,10 @@ describe('SubscriptionManager', () => {
 
     beforeEach(async () => {
         tracker = await startTracker({
-            host: '127.0.0.1',
-            port: trackerPort,
+            listenConfig: {
```

Comment:
maybe this should be `listen` instead of `listenConfig`, I feel like the `config` is implied